test/command/suite/io_flush/recursive/dependent/plugin_register.test

```diff
@@ -0,0 +1,11 @@
+table_create Users TABLE_HASH_KEY ShortText
+
+#@collect-query-log true
+io_flush --recursive dependent
+#@collect-query-log false
+
+plugin_register query_expanders/tsv
+
+#@collect-query-log true
+io_flush --recursive dependent
+#@collect-query-log false
```

Comment:
Why do we need this test?

Reply:
@kou I want to check that `plugin_register` doesn't affect target objects ( before/after `io_flush`,  listed `flush[...]` are same)
Should be removed?

Reply:
`plugin_register` isn't related to `io_flush`.
So I think that this file is needless.